content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -782,6 +782,100 @@ APF adds the following two headers to each HTTP response message.
 - `X-Kubernetes-PF-PriorityLevel-UID` holds the UID of the
   PriorityLevelConfiguration object associated with that FlowSchema.
 
+## Good practices for using API Priority and Fairness
+When a given priority level exceeds its permitted concurrency, requests can 
+experience increased latency or be dropped with an HTTP 429 (Too Many Requests) 
+error. To prevent these side effects of APF, you can modify your workload or 
+tweak your APF settings to ensure there are sufficient seats available to serve
+your requests.
```

Comment:
I think it would be nice to add information about metrics that users should use to observe this issue:
`apiserver_flowcontrol_request_wait_duration_seconds` for increased latency and `apiserver_flowcontrol_rejected_requests_total` for 429. Also, If increased latency/429 is observed across all Priority Levels then most likely it can indicate a different control plane issue. 
Another possibility for some of the users is to horizontally scale kube-apiservers or scale vertically existing kube-apiservers and increase the number of inflight requests.

Reply:
I referenced the following metrics to detect these symptoms of APF:
- apiserver_flowcontrol_rejected_requests_total
- apiserver_flowcontrol_current_inqueue_requests
- apiserver_flowcontrol_request_wait_duration_seconds
- apiserver_flowcontrol_priority_level_seat_utilization

Additionally, I mentioned increasing flag values for max-inflight requests and horizontal scaling as additional options under the "Give more seats to high priority requests" section. To address when APF issues are a symptom of a different root cause, I added this section:
```
Keep in mind that queuing or rejected requests from APF could be induced by 
either an increase in the number of requests or an increase in latency for 
existing requests. For example, if requests that normally take 1s to execute
start taking 60s, it is possible that APF will start rejecting requests because 
requests are occupying seats for a longer duration than normal due to this 
increase in latency. If APF starts rejecting requests across multiple priority 
levels without a significant change in workload, it is possible there is an 
underlying issue with control plane performance rather than the workload or APF 
settings.
```

Reply:
Thanks, looks great.